lib/osf-components/addon/components/search-result-card/styles.scss

```diff
@@ -51,7 +51,7 @@
 }
 
 .withdrawn-label {
-    background-color: $color-bg-gray-darker;
```

Comment:
There is a chance the withdrawn label won't hit a11y standards with this change, depending on the color of the text. Did you test this with the AXE tools?